packages/compass/src/app/index.tsx

```diff
@@ -380,8 +380,8 @@ const app = {
     }
 
     if (
-      semver.gt(APP_VERSION, state.previousVersion) &&
-      state.previousVersion !== DEFAULT_APP_VERSION
+      state.previousVersion !== DEFAULT_APP_VERSION &&
```

Comment:
I'm not sure if we want this change. The idea is that whenever we install an update - even if we downgrade - we'd show the toast that the version changed.  This mirrors the (optional) change [here ](https://github.com/10gen/compass-mongodb-com/pull/117), although in that case it is opt-in for testing only.

At some level it is safe, though since electron/squirrel seems to happily install the latest version even if it is a downgrade.

But I'm totally willing to consider alternatives. We can make this behaviour opt-in through some env var too, but I'd like to try and test the exact same code we release and that a user would run as much as possible.

Reply:
Not a bad idea to change this at all I think. We don't prevent users from manually installing a downgrade and even have some special handling for it in terms of how autoupdate behaves, so for people downgrading manually why not to show that downgrade worked with a toast